elftools/dwarf/locationlists.py

```diff
@@ -8,38 +8,127 @@
 #-------------------------------------------------------------------------------
 import os
 from collections import namedtuple
-
+from ..common.exceptions import DWARFError
 from ..common.utils import struct_parse
 
 LocationExpr = namedtuple('LocationExpr', 'loc_expr')
-LocationEntry = namedtuple('LocationEntry', 'entry_offset begin_offset end_offset loc_expr')
-BaseAddressEntry = namedtuple('BaseAddressEntry', 'entry_offset base_address')
+LocationEntry = namedtuple('LocationEntry', 'entry_offset entry_length begin_offset end_offset loc_expr is_absolute')
+BaseAddressEntry = namedtuple('BaseAddressEntry', 'entry_offset entry_length base_address')
+LocationViewPair = namedtuple('LocationViewPair', 'entry_offset begin end')
 
 class LocationLists(object):
     """ A single location list is a Python list consisting of LocationEntry or
         BaseAddressEntry objects.
+
+        Starting with DWARF5, it may also contain LocationViewPair, but only
+        if scanning the section, never when requested for a DIE attribute.
+
+        The default location entries are returned as LocationEntry with 
+        begin_offset == end_offset == -1
     """
-    def __init__(self, stream, structs):
+    def __init__(self, stream, structs, version = 4, dwarfinfo = None):
```

Comment:
Nit: for default param values no spaces around `=`

Can you document these params in the docstring of the class?